test/js_optimizer/minimal-runtime-emitDCEGraph.js

```diff
@@ -36,7 +36,7 @@ var wasmImports = {
 function run() {
  var ret = _main();
 }
-function initRuntime() {
+function initRuntime(wasmExports) {
```

Comment:
Needed?

Reply:
This just updates the test code to look more like the generated code.  Its good if be testing stuff that looks like that actual output.   This test I assume was originally copied from the output of emscripten and as such it can get stale over time.